service_catalog/api/serializers/instance_serializer.py

```diff
@@ -9,6 +10,7 @@ class InstanceReadSerializer(serializers.ModelSerializer):
     state = serializers.ChoiceField(choices=InstanceState.choices)
     resources = ResourceSerializer(many=True, read_only=True)
     billing_group = BillingGroupInstanceReadSerializer(read_only=True)
+    spoc = UserSerializer(read_only=True)
```

Comment:
This [test ](https://github.com/HewlettPackard/squest/blob/master/tests/test_service_catalog/test_serializers/test_request_serializer.py#L27)to be updated

Reply:
done